mmv1/products/compute/api.yaml

```diff
@@ -15341,17 +15463,16 @@ objects:
           resource: 'RegionSslCertificate'
           imports: 'selfLink'
           description: 'The SSL certificates used by this TargetHttpsProxy'
-     # This field is present in the schema but as of 2019 Sep 23 attempting to set it fails with
-     # a 400 "SSL policy is supported only with global TargetHttpsProxy". jamessynge@ said in an
-     # email sent on 2019 Sep 20 that will support this "in the next 6 months I hope."
-     #- !ruby/object:Api::Type::ResourceRef
-     #  name: 'sslPolicy'
-     #  resource: 'SslPolicy'
-     #  imports: 'selfLink'
-     #  description: |
-     #    A reference to the SslPolicy resource that will be associated with
-     #    the TargetHttpsProxy resource. If not set, the TargetHttpsProxy
-     #    resource will not have any SSL policy configured.
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'sslPolicy'
+        resource: 'RegionSslPolicy'
+        imports: 'selfLink'
+        min_version: beta 
+        description: |
+          A reference to the Region SslPolicy resource that will be associated with
+          the TargetHttpsProxy resource. If not set, the TargetHttpsProxy
+          resource will not have any SSL policy configured.
+     # 2022 May 28 - setSslPolicy not yet defined? 
```

Comment:
Is this what you're looking for? https://cloud.google.com/compute/docs/reference/rest/beta/targetHttpsProxies/setSslPolicy

Reply:
I don't find it under the Region Target HTTPS Proxy methods https://cloud.google.com/compute/docs/reference/rest/beta/regionTargetHttpsProxies
would expect it to be the same as the Target HTTPS Proxy but rather not find out the hard way.